test/unit/rules/custom/test_greater_operator.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
Can we move some of these tests into https://github.com/aws-cloudformation/cfn-lint/blob/main/test/unit/module/custom_rules/test_custom_rules.py

Reply:
Can but not sure if it is a good idea tho, 
https://github.com/aws-cloudformation/cfn-lint/blob/main/test/unit/module/custom_rules/test_custom_rules.py
mostly deal with the rule parser.

While the added test cases are end-to-end assertions based on some template fixtures, which I find similar with rule tests in the same level, e.g.
https://github.com/aws-cloudformation/cfn-lint/blob/main/test/unit/rules/conditions/test_and.py